packages/core/services/FileService/DatabaseFileService/index.ts

```diff
@@ -130,55 +130,81 @@ export default class DatabaseFileService implements FileService {
             .select(annotations.map((annotation) => `"${annotation}"`).join(", "))
             .from(this.dataSourceNames);
 
+        DatabaseFileService.applySelectionFilters(sqlBuilder, selections, this.dataSourceNames);
+
+        return this.handleFileDownload(sqlBuilder.toSQL(), format);
+    }
+
+    /**
+     * Processes selections and applies WHERE clause directly to the SQLBuilder.
+     */
+    public static applySelectionFilters(
+        sqlBuilder: SQLBuilder,
+        selections: Selection[],
+        dataSourceNames: string[]
+    ): void {
+        const subQuerys: string[] = [];
```

Comment:
(nit) subQueries

Reply:
whoops resolved in [089987b](https://github.com/AllenInstitute/biofile-finder/pull/437/commits/089987b35528093503a7ed3ed12286da367f3234)